src/json/test.rs

```diff
@@ -0,0 +1,1460 @@
+extern crate test;
+
+#[cfg(test)]
+use self::Animal::*;
+#[cfg(test)]
+use self::DecodeEnum::*;
+#[cfg(test)]
+use self::test::Bencher;
+#[cfg(test)]
+use {Encodable, Decodable};
+#[cfg(test)]
+use json::json::Json;
+#[cfg(test)]
+use json::json::Json::*;
+#[cfg(test)]
+use json::error::ErrorCode::*;
+#[cfg(test)]
+use json::error::ParserError::*;
+#[cfg(test)]
+use json::error::DecoderError::*;
+#[cfg(test)]
+use json::error::DecoderError;
+#[cfg(test)]
+use json::json::JsonEvent::*;
+#[cfg(test)]
+use json::stack::StackElement::*;
+#[cfg(test)]
+use json::decoder::{Decoder, DecodeResult};
+#[cfg(test)]
+use json::json::JsonEvent;
+#[cfg(test)]
+use json::parser::Parser;
+#[cfg(test)]
+use json::stack::{StackElement, Stack};
+#[cfg(test)]
+use json::encoder::Encoder;
+#[cfg(test)]
+use json::error::EncoderError;
+#[cfg(test)]
+use json::pretty_json::as_pretty_json;
+#[cfg(test)]
+use json::as_json::as_json;
+#[cfg(test)]
```

Comment:
Instead of putting `#[cfg(test)]` on all these imports you should be able to put a `#![cfg(test)]` at the top of this module.